src/main/java/org/isf/exa/gui/ExamBrowser.java

```diff
@@ -235,47 +224,45 @@ public void valueChanged(ListSelectionEvent e) {
-		jButtonDelete.addActionListener(new ActionListener() {
-			public void actionPerformed(ActionEvent event) {
-				if (table.getSelectedRow() < 0) {
-					JOptionPane.showMessageDialog(				
-							ExamBrowser.this,
-	                        MessageBundle.getMessage("angal.common.pleaseselectarow"),
-	                        MessageBundle.getMessage("angal.hospital"),
-	                        JOptionPane.PLAIN_MESSAGE);				
-					return;									
+		jButtonDelete.addActionListener(event -> {
+			if (table.getSelectedRow() < 0) {
+				JOptionPane.showMessageDialog(
+						ExamBrowser.this,
+						MessageBundle.getMessage("angal.common.pleaseselectarow"),
+						MessageBundle.getMessage("angal.hospital"),
+						JOptionPane.PLAIN_MESSAGE);
+				return;
+			}
+			selectedrow = table.convertRowIndexToModel(table.getSelectedRow());
+			Exam examToDelete = (Exam) (model.getValueAt(selectedrow, -1));
+			StringBuilder message = new StringBuilder(MessageBundle.getMessage("angal.exa.deletefolowingexam"))
+					.append(" :")
+					.append("\n")
+					.append(MessageBundle.getMessage("angal.common.code"))
+					.append("= ")
+					.append(examToDelete.getCode())
+					.append("\n")
+					.append(MessageBundle.getMessage("angal.common.description"))
+					.append("= ")
+					.append(examToDelete.getDescription())
+					.append("\n?");
+			int n = JOptionPane.showConfirmDialog(
+					null,
+					message.toString(),
+					MessageBundle.getMessage("angal.hospital"),
+					JOptionPane.YES_NO_OPTION);
+			if ((n == JOptionPane.YES_OPTION)) {
+				boolean deleted;
+
+				try {
+					deleted = manager.deleteExam(examToDelete);
+				} catch (OHServiceException e1) {
+					deleted = false;
+					OHServiceExceptionUtil.showMessages(e1);
 				}
-				selectedrow = table.convertRowIndexToModel(table.getSelectedRow());
-				Exam examToDelete = (Exam)(((ExamBrowsingModel) model).getValueAt(selectedrow, -1));
-				StringBuilder message = new StringBuilder(MessageBundle.getMessage("angal.exa.deletefolowingexam"))
-						.append(" :")
-						.append("\n")
-						.append(MessageBundle.getMessage("angal.common.code"))
-						.append("= ")
-						.append(examToDelete.getCode())
-						.append("\n")
-						.append(MessageBundle.getMessage("angal.common.description"))
-						.append("= ")
-						.append(examToDelete.getDescription())
-						.append("\n?");
-				int n = JOptionPane.showConfirmDialog(
-                        null,
-                        message.toString(),
-                        MessageBundle.getMessage("angal.hospital"),
-                        JOptionPane.YES_NO_OPTION);
-				if ((n == JOptionPane.YES_OPTION)){
-					boolean deleted;
-					
-					try {
-						deleted = manager.deleteExam(examToDelete);
-					} catch (OHServiceException e1) {
-						deleted = false;
-						OHServiceExceptionUtil.showMessages(e1);
-					}
-					
-					if (true == deleted) {
-						reloadTable();
-					}
+
+				if (true == deleted) {
```

Comment:
no need for the `true ==`;  `if (deleted) {` is sufficient and cleaner

Reply:
@dbmalkovsky it's not my code I only moved it to anonymous function and github marked it as green